pyproject.toml

```
@@ -44,6 +44,7 @@ dependencies = [
     "pyyaml>=6.0.2",
     "types-pyyaml>=6.0.12.20240917",
     "pytest-json-report>=1.5.0,<2",
+    "questionary>=2.0.1",
```

We impose a max version on our main dependencies to avoid surprises introduced in a potentially breaking changes (looks like I forgot this when we added `env_init`):
```suggestion
    "pyyaml>=6.0.2,<7",
    "types-pyyaml>=6.0.12.20240917,<7",
    "pytest-json-report>=1.5.0,<2",
    "questionary>=2.0.1,<3",
```